chainer/training/triggers/early_stopping_trigger.py

```diff
@@ -26,22 +32,42 @@ class EarlyStoppingTrigger(object):
             This must be a tuple in the form of ``<int>,
             'epoch'`` or ``<int>, 'iteration'`` which is passed to
             :class:`~chainer.training.triggers.IntervalTrigger`.
-        patients (int) : Counts to let the trigger be patient.
+        patience (int) : Counts to let the trigger be patient.
             The trigger will not fire until the condition is met
-            for successive ``patient`` checks.
+            for successive ``patience`` checks.
         mode (str) : ``'max'``, ``'min'``, or ``'auto'``.
             It is used to determine how to compare the monitored values.
         verbose (bool) : Enable verbose output.
             If verbose is true, you can get more information
         max_trigger: Upper bound of the number of training loops
+
+    .. note::
+       ``patients`` is also available as an alias of ``patience`` for
+       historical reason.
     """
 
     def __init__(self, check_trigger=(1, 'epoch'), monitor='main/loss',
-                 patients=3, mode='auto', verbose=False,
-                 max_trigger=(100, 'epoch')):
+                 patience=None, mode='auto', verbose=False,
+                 max_trigger=(100, 'epoch'), **kwargs):
+
+        # `patients` as an alias of `patience`
+        patients, = argument.parse_kwargs(kwargs, ('patients', None))
+        if patients is None:
+            if patience is None:
+                patience = 3
+            else:
+                pass
+        else:
```

Comment:
How about displaying a warning if `patients` is not `None`?

Reply:
Ditto.